scripts/createYoeoLabelsTORSO.py

```diff
@@ -127,134 +121,129 @@ def range_limited_float_type_0_to_1(arg):
-            mask += ((seg_in == (127, 127, 127)).all(axis=2)).astype(np.uint8)  # Lines
-            mask += (((seg_in == (254, 254, 254)).all(axis=2)).astype(np.uint8) * 2)  # Field
-            seg_out = np.zeros(seg_in.shape, dtype=np.uint8)
-            seg_out[..., 0] = mask
-            seg_out[..., 1] = mask
-            seg_out[..., 2] = mask
-            cv2.imwrite(os.path.join(masks_dir, os.path.splitext(img_name)[0] + ".png"), seg_out)
-        else:
-            print(f"No segmentation found: '{seg_path}'")
-            continue
-
-        images.append(os.path.join(d, "images", img_name))
-        name = os.path.splitext(img_name)[0]  # Remove file extension
-        imgwidth = frame['width']
-        imgheight = frame['height']
-        annotations = []
-
-        for annotation in frame['annotations']:
-            # Ignore if blurred or conceiled and should be ignored
-            if not ((args.ignore_blurred and annotation['blurred']) or
-                (args.ignore_conceiled and annotation['conceiled'])):
-
-                if annotation['type'] in CLASSES['bb_classes']:  # Handle bounding boxes
-                    if annotation['in_image']:
-                        min_x = min(map(lambda x: x[0], annotation['vector']))
-                        max_x = max(map(lambda x: x[0], annotation['vector']))
-                        min_y = min(map(lambda x: x[1], annotation['vector']))
-                        max_y = max(map(lambda x: x[1], annotation['vector']))
-
-                        annowidth = max_x - min_x
-                        annoheight = max_y - min_y
-                        relannowidth = annowidth / imgwidth
-                        relannoheight = annoheight / imgheight
-
-                        center_x = min_x + (annowidth / 2)
-                        center_y = min_y + (annoheight / 2)
-                        relcenter_x = center_x / imgwidth
-                        relcenter_y = center_y / imgheight
-
-                        classID = CLASSES['bb_classes'].index(annotation['type'])  # Derive classID from index in predefined classes
-                        annotations.append(f"{classID} {relcenter_x} {relcenter_y} {relannowidth} {relannoheight}")  # Append to store it later
-                    else:  # Annotation is not in image
-                        continue
-                elif annotation['type'] in CLASSES['segmentation_classes']:  # Handle segmentations
+# Defaults
+create_symlinks = False
+dataset_dir = args.dataset_dir
+destination_dir = args.dataset_dir
+image_names = []  # Collect image paths for train/test split
+
+# Overwrite defaults, if destination path is given
+if args.destination_dir:
+    create_symlinks = args.create_symlinks
+    destination_dir = args.destination_dir
+
+# Create output directories if needed
+images_dir = os.path.join(destination_dir, "images")
+if not os.path.exists(images_dir):
+    os.makedirs(images_dir)
```

Comment:
I think the `if` can be removed if the `exists` flag is used.

Reply:
Yes, we should use the `exist_ok` parameter. Thanks.

Reply:
Nvm this is the correct paramter name.